### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by MichaIng**
Since this is the default README users see when visiting this repository, it makes sense to add information about the current project status.

### CODE REVIEW

**Review by ccrisan — DISMISSED** [review]
LGTM!

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,15 @@
 
 **motionEye** is a web-based frontend for `motion <https://motion-project.github.io>`_. Check out the `wiki <https://github.com/ccrisan/motioneye/wiki>`_ for more details. Changelog is available on the `releases page <https://github.com/ccrisan/motioneye/releases>`_.
 
-Due to personal reasons I can no longer be actively involved with this project. If anyone is interested in taking it over, please contact me and we'll work out together a hand-off plan.
+Due to personal reasons `Calin Crisan <https://github.com/ccrisan>`_ can no longer be actively involved with this project. It has been transferred to the new `motioneye-project <https://github.com/motioneye-project>`_ GitHub organization with a new `developer team <https://github.com/orgs/motioneye-project/people>`_. Please check out the `dev <https://github.com/motioneye-project/motioneye/tree/dev>`_ branch for current development, which includes:
+
+- Migration to Python 3
+- Internationalization
+- Migration to setuptools
+- Many more bug fixes and enhancements
```
Comment (cclauss) [inline README.rst:9]:
```suggestion
- [x] Migration to Python 3
- [x] Internationalization
- [x] Migration to setuptools
- [x] Many more bug fixes and enhancements

```

**Review by cclauss — DISMISSED** [review]

File: README.rst (hunk 1 above)
Comment (MichaIng) [reply README.rst]: Ah dammit, this is not Markdown but reStructuredText, hence the check list cannot be added like that.

Comment (MichaIng) [reply README.rst:9]: Reverted. I guess you wanted a real Markdown checklist. in reStructuredText there is no nativ syntax for it, hence I think a regular list is best.

**Review by MarcelCoding — APPROVED** [review]
The link to the "dev team" is just an empty list if you are not part of the orga.

**Comment by MichaIng** [comment]
> The link to the "dev team" is just an empty list if you are not part of the orga.

Oh, that is bad. Let's see whether we can make this public, if no one has an issue with it?

**Comment by MichaIng** [comment]
Ah right, on the list everyone can select for himself between private and public. I changed it to public now as well, so three public members are shown now. @ccrisan @jmichault @kleini to ping the active members, probably you want to switch to public as well to fill the list for other visitors, but it's perfectly fine to leave this a private choice for everyone, of course 🙂.

Merging now regardless.

**Pull request merged**